example/frog.rkt

```diff
@@ -0,0 +1,42 @@
+#lang racket/base
+
+(require frog/params
+         frog/enhance-body
+         racket/contract
+         rackjure/threading
+         xml/xexpr)
+
+(provide init
+         enhance-body
+         clean)
+
+;; Called early when Frog launches. Use this to set parameters defined
+;; in frog/params.
+(define/contract (init)
+  (-> any)
+  (current-scheme/host "http://www.example.com")
+  (current-title "My Blog")
+  (current-author "The Unknown Author"))
+
+;; Called once per post and non-post page, on the contents.
+(define/contract (enhance-body xs)
+  (-> (listof xexpr/c) (listof xexpr/c))
+  ;; Here we pass the xexprs through a series of functions.
+  (~> xs
+      (syntax-highlight #:python-executable "python"
```

Comment:
> How about making "python" the keyword default value?

OK. Done in https://github.com/greghendershott/frog/pull/194/commits/27a3af4fd3c33834ff97f2a145980f9d38981ae9#diff-758af76c72ceeace13c505f9d6419389R25